### DISCUSSION TITLE
Create an animated loading or creation state

### DISCUSSION THREAD

**Discussion by gdoyle1** [patternfly/patternfly#5462] [Feature requests]
**High priority**

We have a few different data loading components available today: spinner and skeleton loading. These are sometimes difficult to use on pages where we aren't sure of what the data will look like and it is too long of a loading state to use a spinner. Another use case is for creation states - some products use building-centric full-screen animations to demonstrate something is being created (where the spinner and skeleton loading patterns would not work).

It would be nice to have an animated component/demo for a page loading or creation state. Some ideas that came out of the animation working group were to possibly use the product's logo to do so. Google does this pretty well already, so might be worth looking there!

**Comment by ghost** [comment]
i want to work on this....
